packages/ui/src/components/DataStory/clients/JsClient.tsx

```diff
@@ -30,20 +30,18 @@ export class JsClient implements ServerClient {
         atNodeId,
         limit = 10,
         offset = 0,
-      }: {
-        atNodeId: string,
-        limit?: number,
-        offset?: number,
-
-      }) => {
-        if(!this.executor) return []
+      }: TableItems) => {
+        if(!this.executor) return { items: [], total: 0 };
 
         const items = this.executor.storage.items.get(atNodeId) || [];
 
-        return items.slice(offset, offset + limit);
+        return {
```

Comment:
This method have some typing issue, originating in Storage interface:

```
export interface Storage {
  items: Map<NodeId | LinkId, ItemValue>;
}
```

the Map value should be `ItemValue[]` here, am I right? Please address that, and dependent types.

Reply:
Very good 👍